### PR TITLE
feature/github_templates

### PR DESCRIPTION
**Pull request by danieldeveloper001**
## Description

:sparkles: Added bug and feature request templates

## Reason

- Resolves issue #16 

## Other
No additional details.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report-template.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+## Steps
+
+### Step 1
+Please, provide here a detailed description about the step 1.
+
+### Step 2
+Please, provide here a detailed description about the step 2.
+
+### Step 3
```
[the remark this answers is not in the document]
Comment (danieldeveloper001) [reply .github/ISSUE_TEMPLATE/bug-report-template.md]: The steps were just a suggestion, something like a placeholder that one can use as a base template. Do you mean something like this?

## Steps

- Step description here
- Step description here
- Step description here

... and so on?

File: .github/ISSUE_TEMPLATE/bug-report-template.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+The expected behaviour is <provide here detailed description and, if necessary, screenshots>.
+
+## Actual Behavior
+The actual behaviour is <provide here detailed description and, if necessary, screenshots>.
+
+## Environment
+- Device Type: (e.g.: Desktop or Mobile)
+- Operating System: (e.g.: Microsoft Windows or Ubuntu Linux)
+  - Version: (e.g.: 10 or 18.04)
+- Browser: (e.g.: Mozilla Firefox or Google Chrome)
```
[the remark this answers is not in the document]
Comment (danieldeveloper001) [reply .github/ISSUE_TEMPLATE/bug-report-template.md]: Sorry, my bad, removing browser from the template.

[the remark this answers is not in the document]
Comment (danieldeveloper001) [reply .github/ISSUE_TEMPLATE/bug-report-template.md]: Do you think the Python version is relevant when submitting the bug?

File: .github/ISSUE_TEMPLATE/bug-report-template.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+name: Bug Report Template
+about: A simple bug report template.
+
+---
+
+## Description
+
+Please, provide here a simple description about the bug.
+
+## Steps
```
[the remark this answers is not in the document]
Comment (danieldeveloper001) [reply .github/ISSUE_TEMPLATE/bug-report-template.md]: Changing the title from "steps" to "steps to reproduce"

File: .github/ISSUE_TEMPLATE/bug-report-template.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+---
+name: Bug Report Template
+about: A simple bug report template.
+
+---
+
+## Description
+
+Please, provide here a simple description about the bug.
```
[the remark this answers is not in the document]
Comment (danieldeveloper001) [reply .github/ISSUE_TEMPLATE/bug-report-template.md]: Grammar fixed.

File: .github/ISSUE_TEMPLATE/bug-report-template.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+Please, provide here a detailed description about the step 1.
+
+### Step 2
+Please, provide here a detailed description about the step 2.
+
+### Step 3
+Please, provide here a detailed description about the step 3.
+
+## Expected Behavior
+The expected behaviour is <provide here detailed description and, if necessary, screenshots>.
```
[the remark this answers is not in the document]
Comment (danieldeveloper001) [reply .github/ISSUE_TEMPLATE/bug-report-template.md]: Grammar fixed.

File: .github/ISSUE_TEMPLATE/bug-report-template.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+## Environment
+- Device Type: (e.g.: Desktop or Mobile)
+- Operating System: (e.g.: Microsoft Windows or Ubuntu Linux)
+  - Version: (e.g.: 10 or 18.04)
+- Browser: (e.g.: Mozilla Firefox or Google Chrome)
+  - Version: (e.g.: 1, 2, 3)
+
+## Other
+Please, provide here additional details that can be helpful.
```
[the remark this answers is not in the document]
Comment (danieldeveloper001) [reply .github/ISSUE_TEMPLATE/bug-report-template.md]: Grammar fixed.

File: .github/ISSUE_TEMPLATE/bug-report-template.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+---
+
+## Description
+
+Please, provide here a simple description about the bug.
+
+## Steps
+
+### Step 1
+Please, provide here a detailed description about the step 1.
```
[the remark this answers is not in the document]
Comment (danieldeveloper001) [reply .github/ISSUE_TEMPLATE/bug-report-template.md]: Sentence changed, yours looks better.

**Comment by danieldeveloper001** [comment]
@GhostofGoes, I've made some changes, based on your feedback.
Please, let me know if the templates are more appropriate now :wink:

**Comment by heavysaturn** [comment]
This is pretty good. I'd like to see something on the feature request templates about the difficulty of implementing this, and perhaps something about whether or not it would require third party modules and if so, which ones.

**Comment by danieldeveloper001** [comment]
@heavysaturn, should we close it due to the same reason as stated [here](https://github.com/discord-python/hacktoberbot/issues/13#issuecomment-427561408)?

**Comment by heavysaturn** [comment]
no, that is fine. there is an issue for it and nobody else is working on it. you are welcome to work on it if you want.

**Comment by heavysaturn** [comment]
Actually I'm just gonna merge this. Thanks for the contribution, @danieldeveloper001.

**Pull request merged**